components/Workspaces/WorkspaceLayout.tsx

```diff
@@ -77,6 +81,16 @@ export const WorkspaceLayout = ({ workspaceId, banner, children, footer }: Works
           <div className="border-t flex">{footer}</div>
         </div>
       ) : null}
+      {!user ? (
+        <div className="sticky bottom-0 bg-light-slate-2 border-t h-16 pr-3 flex flex-col justify-center items-end">
+          <p className="w-full px-4">
+            <Link href="/start" className="text-sauced-orange hover:underline">
+              Sign up
+            </Link>{" "}
+            to start tracking your projects!
+          </p>
+        </div>
+      ) : null}
```

Comment:
This will center the nudge banner, and it will also only show it on small screens as per the issue.

```suggestion
      {!user ? (
        <div className="sticky bottom-0 bg-light-slate-2 border-t h-16 pr-3 grid place-content-center px-4 md:hidden">
          <span>
            <Link href="/start" className="text-sauced-orange hover:underline">
              Sign up
            </Link>{" "}
            to start tracking your projects!
          </span>
        </div>
      ) : null}
```

**Before**

![CleanShot 2024-04-25 at 18 36 33](https://github.com/open-sauced/app/assets/833231/40478102-6683-4710-b8ab-8f7be8793f71)

**After**

![CleanShot 2024-04-25 at 18 35 49](https://github.com/open-sauced/app/assets/833231/de8c9de5-a9f3-424b-827d-7669f1dd7fc7)

Looking at the issue, there's an X to dismiss. Is that just from the mockup @brandonroberts @isabensusan, or do we want them to be able to dismiss the banner? If so we can use `useLocalStorage` for that like we do in other places.